gui/slick/views/displayShow.mako

```diff
@@ -540,8 +493,13 @@ $(document).ready(function(){
                     ${file_size}
                 % endif
             </td>
+            ## <% date = (sbdatetime.sbdatetime.sbfdate(sbdatetime.sbdatetime.convert_to_setting(network_timezones.parse_date_time(epResult['airdate'], show.airs, show.network))), 'never')[int(epResult['airdate']) == 1] %>
+            <% date = sbdatetime.sbdatetime.convert_to_setting(network_timezones.parse_date_time(epResult['airdate'], show.airs, show.network)) %>
```

Comment:
@MGaetan89 any chance we could get some date/time formatting functions added in python so we don't have to do this every time?